package.json

```diff
@@ -8,7 +8,10 @@
         "test": "jest --runInBand tests/**/*.test.ts",
         "test:postgres": "jest --runInBand tests/postgres/*.test.ts tests/*.test.ts",
         "test:clickhouse": "jest --runInBand tests/clickhouse/*.test.ts",
-        "benchmark": "node --expose-gc node_modules/.bin/jest --runInBand benchmarks/",
+        "benchmark": "yarn run benchmarks:clickhouse && yarn run benchmark:postgres && yarn run benchmarks:vm",
+        "benchmark:clickhouse": "node --expose-gc node_modules/.bin/jest --runInBand benchmarks/vm/",
+        "benchmark:postgres": "node --expose-gc node_modules/.bin/jest --runInBand benchmarks/clickhouse/",
+        "benchmark:vm": "node --expose-gc node_modules/.bin/jest --runInBand benchmarks/postgres/",
```

Comment:
This looks like benchmark:postgres for instance runs ClickHouse benchmarks, am I missing the point? 🤔

Reply:
ugh, something got mangled here 😊